history/cultures/naga.txt

```diff
@@ -37,6 +38,8 @@
 	discover_innovation = innovation_hereditary_rule 
 	discover_innovation = innovation_royal_prerogative 
 	#
+	discover_innovation = innovation_guerilla_warfare
```

Comment:
Why? They lived in the ocean for 10000 years.

Reply:
For those living on land, they would adapt no? They are qualified for the tech so if you want me exclude them lmk